extensions/guacamole-auth-sso/modules/guacamole-auth-sso-openid/src/main/java/org/apache/guacamole/auth/openid/conf/ConfigurationService.java

```diff
@@ -126,6 +128,12 @@ public class ConfigurationService {
 
     };
 
+    private static final StringGuacamoleProperty OPENID_ATTRIBUTES_CLAIM_TYPE =
+            new StringGuacamoleProperty() {
+                @Override
+                public String getName() { return "openid-attributes-claim-type"; }
+            };
```

Comment:
It would be better to use `StringListProperty` for this, rather than manually parse the string into an array each time a claim is being handled.